### ISSUE TITLE
Deprecate contact page and move contact/website page to contact page

### ISSUE THREAD

**Issue by danieljames-dj** [thewca/worldcubeassociation.org#8665]
The contact page (https://www.worldcubeassociation.org/contact) doesn't have any useful data that is not available in teams-committees page or website contact form (https://www.worldcubeassociation.org/contact/website). So I guess it will be good to delete the contact page, and move the contact/website page to contact route.

Before making the change, we will be needing approval on following three things:

1. Whether the description of each team/committee should be shown anywhere else or combined with the ones in team-committee page.
2. Whether it's fine to remove /contact page considering only usage is linked from home page, which can be redirected to contact/website page.
3. Whether moving contact/website to contact will be good, which means if we go to <website_url>/contact, it will open the contact form directly.

**Comment by dunkOnIT** [comment]
I agree with only having one contact page, and not duplicating information that is displayed about teams. The question then is how do people contact teams - I see two options:

## Option 1 - Contact Form
- have a "Contact a team" radio button `/contact/website`
- When "Contact a team" is selected, we give a dropdown that allows for different teams to be selected - when a team is selected, a description of the team (ideally the one used on the `teams-committees` page) is displayed 

## Option 2 - Teams Page
- Emails are already displayed there - we just leave it as-is and remove `/contact`? Only downside is that it's not necessarily intuitive to users that the contact information for teams would be found on Teams/Committees, not on Contact
- We could have a `Looking to contact a team? Find their email on the Teams page` starburst on `/contact` that would resolve that confusion?
